js/dataframe/shared/EditableCell.svelte

```diff
@@ -93,6 +97,10 @@
 		background: transparent;
 	}
 
+	span.focused {
```

Comment:
Moved the box-shadow from the `<td>` to the `<span>` so that it is visible even when there is a background-color